### DISCUSSION TITLE
Are there any facade patterns written for video js?

### DISCUSSION THREAD

**Discussion by gigaboy** [videojs/video.js#8084] [Q&A]
Or is one even needed?
I want to embed YouTube videos.

**Comment by amtins** [comment]
@gigaboy did you try the [videojs-youtube](https://github.com/videojs/videojs-youtube) plugin ?

**Comment by gigaboy** [comment]
I just built two simple pages testing the technology using 

https://github.com/justinribeiro/lite-youtube (built for implementing facades)
and
Video.js with the youtube plugin.

Ran both through gtmetrix, screenshots attached.

https://www.rosemariespeaks.com/z/vjs.html
and
https://www.rosemariespeaks.com/z/lyte.html

GTMetrix complains that façade is needed with the vjs version.  I would prefer to use the YT plugin as Video.js provides much more customization needed for my particular app.

Any insights?

THANKS!
Mark
